app/screens/ScheduleScreen/ScheduleScreen.tsx

```diff
@@ -227,43 +177,14 @@ export const ScheduleScreen: React.FC<TabScreenProps<"Schedule">> = () => {
             decelerationRate="fast"
             renderItem={({ index, item: schedule }) => (
               <View style={[$container, { width }]}>
-                <FlashList
+                <FlashList<ScheduleCardProps>
```

Comment:
@robinheinze I tried to see if I could separate this `FlashList` out of the screen and make it a component as well but I think it makes sense to leave it here since the `Animated.FlatList` and this `FlashList` together handles the scrolling animation vertically and horizontally and the screen itself should handle the animation and the logic I think but I'm not 100%. Let me know what you think :)

Reply:
Agreed that it should stay in this screen! Everything else you've done already does a LOT to clean things up. I still think there's an opportunity here to improve readability though, and pull the `renderItem` into a const (with `useCallback`) outside of the return block